src/Sentry.Unity/Integrations/UnityBeforeSceneLoadIntegration.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Collections.Generic;
+using Sentry.Integrations;
+using UnityEngine.SceneManagement;
+
+namespace Sentry.Unity.Integrations
+{
+    internal sealed class UnityBeforeSceneLoadIntegration : ISdkIntegration
+    {
+        public void Register(IHub hub, SentryOptions options)
+        {
+            var data = SceneManager.GetActiveScene().name is { } name
+                ? new Dictionary<string, string> {{"scene", name}}
+                : null;
+
+            SentrySdk.AddBreadcrumb("BeforeSceneLoad", data: data);
+
+            options.DiagnosticLogger?.Log(SentryLevel.Debug, "Complete Sentry SDK initialization.");

```

Comment:
This log message is confusing, what about:

```suggestion
            options.DiagnosticLogger?.Log(SentryLevel.Debug, "Registered BeforeSceneLoad integration.");
```

Arent' we supposed to hook into an event to get notified when a scene loads?
This will register the scene name when the SDK initializes and never again, right?